src/components/designer/SyncButton.tsx

```diff
@@ -17,9 +17,10 @@ const Styled = {
 
 interface Props {
   network: Network;
+  children?: ReactNode;
```

Comment:
It looks like `children` is leftover from your prior changes and is not needed any longer.